### ISSUE TITLE
Baseline FCNs

### ISSUE THREAD

**Issue by vhrabar** [vhrabar/fcn-morphology-isic2018#3]
Implement baseline FCNs with Resnet-50 and 101 backbone provided by the `Torchvision` library

**Scope**
- Training Loop
- Base models

**Acceptance Criteria**
- Implemented:
    - [ ] 0-variant (FCN-ResNet50 backbone, trainable, producing valid segmentation masks)
    - [ ]  101-variant (FCN-ResNet101 backbone, trainable, producing valid segmentation masks)
 - [ ] Training loop correctly performs forward/backward/optimizer step
